.github/workflows/ci.yml

```diff
@@ -704,19 +701,19 @@ jobs:
               config: default
             - suite: suite-azure
               ignore exclusion if: >-
-                ${{ needs.build-pt.outputs.have_azure_secrets == 'true' }}
+                ${{ needs.check-secrets.outputs.have_azure_secrets == 'true' }}
 
             - suite: suite-gcs
               config: default
             - suite: suite-gcs
               ignore exclusion if: >-
-                ${{ needs.build-pt.outputs.have_gcp_secrets == 'true' }}
+                ${{ needs.check-gcp-secrets.outputs.have_gcp_secrets == 'true' }}
 
             - suite: suite-delta-lake-databricks
               config: hdp3
             - suite: suite-delta-lake-databricks
               ignore exclusion if: >-
-                ${{ needs.build-pt.outputs.have_databricks_secrets == 'true' }}
+                ${{ needs.check-databricks-secrets.outputs.have_databricks_secrets == 'true' }}
```

Comment:
The outcome in this case a `boolean` right?

(`ignore exclusion if` is set to `"false"`)

Reply:
No, it's a string. I think it might be caused by using the multiline block (`>-` where `>` is replacing newlines with spaces and `-` chops the trailing newline).

You can see the final yaml after evaluating the GHA expressions in the step that writes it into the file:
![image](https://user-images.githubusercontent.com/795177/196162206-06ee66f1-091c-4211-95c1-8debc7c34341.png)

and then you can see how it gets read into Python in the next step:
![image](https://user-images.githubusercontent.com/795177/196162287-5463ab46-6b48-492f-9f26-ca6208e8079b.png)